tests/detectors/test_detectors_shields.py

```diff
@@ -31,7 +31,9 @@ def test_up_detector_default_strings(up_detector):
     ]
 
     for response in default_responses:
-        attempt = Attempt(prompt="test", lang=up_detector.lang_spec.split(",")[0])
+        attempt = Attempt(
+            prompt=Message("test"), lang=up_detector.lang_spec.split(",")[0]
```

Comment:
`lang` should be passed to the `Message` constructor:
```suggestion
            prompt=Message(text="test", lang=up_detector.lang_spec.split(",")[0])
```

Applies to some other locations in the test as well, removal from the `Attempt` constructor should make identify tests that need to refactor.